debian/postinst

```diff
@@ -7,6 +7,7 @@ set -e
 ESM_KEY_ID_TRUSTY="56F7650A24C9E9ECF87C4D8D4067E40313CB4B13"
 ESM_APT_GPG_KEY="/etc/apt/trusted.gpg.d/ubuntu-esm-v2-keyring.gpg"
 ESM_APT_GPG_KEY_OLD="/etc/apt/trusted.gpg.d/ubuntu-esm-keyring.gpg"
+ESM_APT_SOURCE_FILE_PRECISE="/etc/apt/sources.list.d/ubuntu-esm-precise.list"
 ESM_APT_SOURCE_FILE_OLD="/etc/apt/sources.list.d/ubuntu-esm-trusty.list"
```

Comment:
Do we want to rename the `_OLD` variables, as the precise versions of those files are also "old", just in a different way?

Reply:
done. ESM_INFRA_* for the win. dropping _OLD